File: archive/cluster_indices/populate_inputs.m
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% NAME:	populate_inputs.m
%% PROJECT: 	Survival Curves for Different Cancer Types
%% AUTHOR:	Yugarshi Mondal
%% DESCRIPTION: This script takes CNN outputs which have already been thresholded and creates csvs
%%		that detect lymp infiltration.
%% INPUTS:	
%% OUTPUTS:

%addpath('/data02/shared/yoshi/u24_lymphocyte/lym_outputs')
%% Relative Path to get_patch_til_svs_wrap
addpath('../lym_outputs')

%%%%%%%%%%%%%%%%%% Edit these paths %%%%%%%%%%%%%%%%%%%%%%
processedImages = '/data08/shared/lehhou/active_learning_osprey';
outputs = '/data02/shared/yoshi/u24_lymphocyte/cluster_indices/inputs';
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


cancer_types = {'blca','brca','cesc','coad','luad','lusc','paad','prad','read','skcm','stad','ucec','uvm'};

for i = 1:length(cancer_types)
	i
	mkdir(strcat('./inputs/', cancer_types{i}));
	get_patch_til_svs_file_wrap(cancer_types{i},outputs,processedImages);
end

File: cluster_indices/populate_inputs.py
import os
import sys
from get_patch_til_svs_file_wrap import get_patch_til_svs_file_singlefile_wrap

cancer_types = ['read', 'uvm'];

processedImages = sys.argv[1];
outputs = sys.argv[2];

for ctype in cancer_types:
    os.makedirs(os.path.join(outputs, ctype));
    get_patch_til_svs_file_singlefile_wrap(ctype, outputs, processedImages);

